rogue-thi-app/pages/food.js

```diff
@@ -218,10 +237,29 @@ export default function Mensa () {
       </AppNavbar>
 
       <AppBody>
-        <ReactPlaceholder type="text" rows={20} ready={foodDays}>
-          <SwipeableTabs className={styles.tab}>
-            {foodDays && foodDays.map((day, idx) => renderMealDay(day, idx))}
-          </SwipeableTabs>
+        <div className={styles.weekSelector}>
+          <Button className={styles.prevWeek} variant="link" onClick={() => setPage(0)} disabled={page === 0}>
+            <FontAwesomeIcon title="Woche zurück" icon={faChevronLeft} />
+          </Button>
+          <div className={styles.currentWeek}>
+            {page === 0 && WORD_THIS_WEEK}
+            {page === 1 && WORD_NEXT_WEEK}
+          </div>
+          <Button className={styles.nextWeek} variant="link" onClick={() => setPage(1)} disabled={page === 1}>
```

Comment:
If there is no data available only for next week, it is still possible to switch to the empty next week. I think we should disable this button then.